### PR TITLE
Support for basic line charts

### PR DESCRIPTION
**Pull request by collinjackson**

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
probably don't need to land the tests file if it's empty.  lgtm

File: lib/src/base.dart [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    super.paint(canvas, offset);
+  }
+}
+
+class ChartPainter {
+  ChartPainter(this.data);
+
+  ChartData data;
+
+  Point _convertPointToRectSpace(sky.Point point, Rect rect) {
```
Comment (abarth) [inline lib/src/base.dart]: s/sky.Point/Point/ should work if you prefer.

**Comment by abarth** [comment]
LGTM
